core/src/wallet/ethereum/explorers/LedgerApiEthereumLikeBlockchainExplorer.cpp

```diff
@@ -119,7 +119,8 @@ namespace ledger {
         LedgerApiEthereumLikeBlockchainExplorer::getTransactions(const std::vector<std::string> &addresses,
                                                         Option<std::string> fromBlockHash,
                                                         Option<void *> session) {
-            return getLedgerApiTransactions(addresses, fromBlockHash, session);
+            bool isSnakeCase = true;
```

Comment:
I did that to avoid going back and forth between files to check the meaning of `true` passed parameter, but if you feel like it's not necessary then I'll remove it